protocol/contracts/leaser/src/leaser.rs

```diff
@@ -241,4 +324,8 @@ mod test {
             reserve: migration_spec,
         }
     }
+
+    fn migrate_msg() -> impl Fn(Addr) -> MigrateMsg {
+        |_customer| MigrateMsg {}
+    }
```

Comment:
As mentioned above, it's going to be more compact and understandable with less jumping around if it's inlined.

Reply:
frankly, I do not understand what you mean